### PR TITLE
add link to video from EMBO workshop

### PR DESCRIPTION
**Pull request by haesleinhuepf**
Hi all,

I'm proposing another link for the list of workshops. 

Btw. I'm wondering is this the right page that is rendered here?
https://napari.org/stable/further-resources/napari-workshops.html

That one seems out of sync.

Thanks!
Best,
Robert

### CODE REVIEW

**Comment by Czaki** [comment]
This is current page: https://napari.org/dev/further-resources/napari-workshops.html

We have a plan to work on updating part of the webpage without releasing napari. but it is not done yet.

**Review by Czaki — APPROVED** [review]

File: docs/further-resources/napari-workshops.md [hunk 1]
```diff
@@ -21,7 +21,9 @@ or contact the core developers on [zulip chat](https://napari.zulipchat.com/logi
 ## Workshops
 
 *Workshops are listed from newest to oldest.*
-* September 2023, [plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
+* September 2023
+  * [plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
+  * [Bioimage Analysis with Python and Napari (video lecture)](https://www.youtube.com/watch?v=QDS5t7oZH-c) at the EMBO Practical Course for Advanced Methods in Bioimage Analysis
```
Comment (psobolewskiPhD) [inline docs/further-resources/napari-workshops.md]:
```suggestion
  * [Bioimage Analysis with Python and Napari (video lecture)](https://www.youtube.com/watch?v=QDS5t7oZH-c) at the [EMBO Practical Course for Advanced Methods in Bioimage Analysis](https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/) 
```
I think that's the proper link for anyone that wants a bit more background about the video.

**Review by psobolewskiPhD — CHANGES_REQUESTED** [review]
I think it would be nice to link to the page of the course/embl -- did I get the link right @haesleinhuepf ?

**Comment by haesleinhuepf** [comment]
> I think it would be nice to link to the page of the course/embl -- did I get the link right @haesleinhuepf ?

yes, great hint @psobolewskiPhD !

File: docs/further-resources/napari-workshops.md [hunk 2]
```diff
@@ -21,7 +21,9 @@ or contact the core developers on [zulip chat](https://napari.zulipchat.com/logi
 ## Workshops
 
 *Workshops are listed from newest to oldest.*
-* September 2023, [plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
+* September 2023
+  * [plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
```
Comment (psobolewskiPhD) [inline docs/further-resources/napari-workshops.md]:
```suggestion
  * [Plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
```

File: docs/further-resources/napari-workshops.md [hunk 3]
```diff
@@ -21,7 +21,9 @@ or contact the core developers on [zulip chat](https://napari.zulipchat.com/logi
 ## Workshops
 
 *Workshops are listed from newest to oldest.*
-* September 2023, [plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
+* September 2023
+  * [Plugin design](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/design23.html) and [software development workshop](https://chanzuckerberg.github.io/napari-plugin-accel-workshops/workshops/softwaredev.html) for [napari plugin grantees](https://chanzuckerberg.com/rfa/napari-plugin-grants/).
+  * [Bioimage Analysis with Python and Napari (video lecture)](https://www.youtube.com/watch?v=QDS5t7oZH-c) at the [EMBO Practical Course for Advanced Methods in Bioimage Analysis](https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/)
```
Comment (psobolewskiPhD) [inline docs/further-resources/napari-workshops.md]:
```suggestion
  * [Bioimage Analysis with Python and Napari (video lecture)](https://www.youtube.com/watch?v=QDS5t7oZH-c) at the [EMBO Practical Course for Advanced Methods in Bioimage Analysis](https://www.embl.org/about/info/course-and-conference-office/events/bia23-01/)  
  
```

**Review by psobolewskiPhD — APPROVED** [review]
Made a tiny tweak: capitol P and some an empty line

**Comment by psobolewskiPhD** [comment]
Hmm...something is up with the CircleCI previews...
I'm getting
```
{"message":"not found"}
```
Here's what the downloaded artifact looks like:
image

**Comment by psobolewskiPhD** [comment]
> Hmm...something is up with the CircleCI previews...

This has been resolved by https://github.com/napari/docs/pull/278

**Pull request merged**